tox.ini

```diff
@@ -19,7 +19,7 @@ commands =
     # -l: show locals in tracebacks
     # --tb=short: short traceback print mode
     # --strict: marks not registered in configuration file raise errors
```

Comment:
seems like we need the comment 

`# --ignore=tests/integration: exclude integration tests`

Reply:
added.